### PR TITLE
doc: Update images in documentation

### PR DESCRIPTION
**Pull request by nejraselimovic**
#### Summary
Proposal for images adjustment as part of the issue #127.

#### Changes
Images dimensions are manually adjusted to appear better on the docs page.

#### Checklist
- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/gateways/ciscowirelessgateway/_index.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ This page guides you to connecting the Cisco Wireless Gateway for LoRaWAN to {{%
 
 Technical specifications for this gateway can be found in [Cisco's official documentation](https://www.cisco.com/c/en/us/products/routers/wireless-gateway-lorawan/). 
 
-![Cisco LoRaWAN Gateway](cisco.jpg)
+<p align="center">
+  <img width="55%" src="cisco.jpg" alt="Cisco LoRaWAN Gateway" >
+</p>
```
Comment (johanstokking) [inline doc/content/gateways/ciscowirelessgateway/_index.md:15]: Why not use `figure` here?

File: doc/content/gateways/dragino-lps8/_index.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ The **Dragino LPS8** is a LoRaWAN indoor gateway based on the OpenWrt Linux syst
 
 This page guides you on how to connect the Dragino LPS8 to {{% tts %}}.
 
-{{< figure src="dragino-lps8.jpg" alt="Dragino LPS8" >}}
+{{< figure src="dragino-lps8.jpg" alt="Dragino LPS8" width=50% >}}
```
Comment (johanstokking) [inline doc/content/gateways/dragino-lps8/_index.md:13]: Need to put `50%` in quotes?

File: doc/content/gateways/laird-sentrius/_index.md [hunk 3]
```diff
@@ -3,15 +3,17 @@ title: "Laird Sentrius™ RG1xx LoRaWAN Gateway"
...
 
 This page contains information about connecting the Laird Sentrius™ RG1xx to {{% tts %}}
 
 <!--more-->
 
 The technical specifications can be found in [Laird's official documentation](https://www.lairdconnect.com/wireless-modules/lorawan-solutions/sentrius-rg1xx-lorawan-gateway-wi-fi-ethernet-optional-lte-us-only#documentation). The Laird Sentrius™ RG1xx LoRaWAN Gateway supports two ways to connect with {{% tts %}}, using either the Semtech Packet Router or LoRa Basic Station.
 
-![Laird RG1xx](laird.png)
+<p align="center">
+  <img width="65%" src="laird.png" alt="Laird RG1xx" >
```
Comment (johanstokking) [inline doc/content/gateways/laird-sentrius/_index.md:15]: Like here

File: doc/content/gateways/laird-sentrius/_index.md [hunk 4]
```diff
@@ -25,15 +27,15 @@ Create a gateway by following the instructions for the [Console]({{< ref "/getti
 
 The **EUI** of the gateway can be found on the configuration web page of the gateway as **Gateway ID**, or on the bottom label as **DevEUI**.
 
-{{< figure src="eui-small.png" alt="Gateway EUI" >}}
+{{< figure src="eui-small.png" alt="Gateway EUI" width=50% >}}
```
Comment (johanstokking) [inline doc/content/gateways/laird-sentrius/_index.md:30]: Idem

File: doc/content/gateways/mikrotikrouterboard/_index.md [hunk 5]
```diff
@@ -20,7 +20,7 @@ There are multiple interfaces to configure the gateway parameters. This page gui
 
 The MikroTik Routerboard wAP LoRa8 kit comes with a PoE adapter. The following image shows the proper connections to the device.
 
-{{< figure src="mikrotik-routerboard-connections.jpg" alt="MikroTik Routerboard wAP LoRa8 kit connections" >}}
+{{< figure src="mikrotik-routerboard-connections.jpg" alt="MikroTik Routerboard wAP LoRa8 kit connections" width=80% >}}
```
Comment (johanstokking) [inline doc/content/gateways/mikrotikrouterboard/_index.md:23]: I'm seeing 45, 50, 55, 65, 70, 80 and 85% in this PR. Is this a percentage of screen width or of the original image? Is there a guideline for the right number?

**Review by benolayinka — COMMENTED** [review]
I also don't like the `figure` shortcode, and it's bitten us before because we can't check if the source is valid (hugo still builds with missing `figure` images.)

There is a hugo feature which allows us to use the markdown image syntax to render our own image tags, so `image-title` -> https://gohugo.io/getting-started/configuration-markup/#image-markdown-example

@nejraselimovic can you use that to replace our existing tags with something you standardize, like your
```
<p align="center">
  <img width="55%" src="cisco.jpg" alt="Cisco LoRaWAN Gateway" >
</p>
```

It looks like you can also use Hugo's image processing to automatically scale images to fit a certain width or height -> https://gohugo.io/content-management/image-processing/#fit

File: doc/content/gateways/dragino-lps8/_index.md (hunk 2 above)
Comment (nejraselimovic) [reply doc/content/gateways/dragino-lps8/_index.md:13]: No, it doesn't have to be in quotes

**Comment by benolayinka** [comment]
> I also don't like the `figure` shortcode, and it's bitten us before because we can't check if the source is valid (hugo still builds with missing `figure` images.)
> 
> There is a hugo feature which allows us to use the markdown image syntax to render our own image tags, so `image-title` -> https://gohugo.io/getting-started/configuration-markup/#image-markdown-example
> 
> @nejraselimovic can you use that to replace our existing tags with something you standardize, like your
> 
> ```
> <p align="center">
>   Cisco LoRaWAN Gateway
> </p>
> ```
> 
> It looks like you can also use Hugo's image processing to automatically scale images to fit a certain width or height -> https://gohugo.io/content-management/image-processing/#fit

This will also close https://github.com/TheThingsIndustries/lorawan-stack-docs/issues/83

**Comment by nejraselimovic** [comment]
@johanstokking I used `img` instead of `figure` because with `figure` I wasn't sure how to remove border shadow on some images that looked nicer without it (example below), while with `img` it was easy peasy. Anyways, I managed to figure it out at the end - I modified css so you can use `figure` and add a `class` attribute to render the image without this box shadow. 
images-example

@benolayinka If we want to go that way, I can try working on it ofc

**Comment by nejraselimovic** [comment]
I'm not really sure how to approach the width/size problem. For example, we have these small images presenting gateways, devices, whatever, and on the other hand, for some stuff we have screenshots of the whole screen. Also, we want to crop some of those screenshots because the text is unreadable or simply there's too much (unnecessary) info. I'm not sure they can render nicely with a default figure width, if their size is variable. I'll keep working on it, but feel free to throw a suggestion if you have one

**Review by pierrephz — APPROVED** [review]

**Comment by kschiffer** [comment]
If we just use relative width values (like `80%`) these will be ok on desktop, but then they will be unnecessarily shrunk down on mobile screen sizes. So even better would be `max-width` and absolute values, but that would likely not work with plain markdown.

One idea is to distinguish between screenshots and inline images and use two distinct stylings for those.

Sreenshots:

*   Should fill the whole width of the container
*   They will be rendered in a bordered box
*   They can be clicked and viewed in full, if whished
*   Should be cropped (if necessary) so that the relevant area is readable
*   They will use a `max-height`, so that e.g. screenshots of mobile screens will appear with reasonable size
    *   The container will still occupy 100% of the width then but the image will be centered

Inline images:

*   Use absolute size values (which have to be set individually, every time)
*   Will shrink in smaller screen sizes (as soon as they fill 100% of the parent container)
*   Do not have any size restrictions (but images used here should be prepared properly)

**Comment by kschiffer** [comment]
I'll take a look at adding the styles for this.

**Comment by nejraselimovic** [comment]
> I'll take a look at adding the styles for this.

That would be perfect! I wasn't sure how to wrap this up and I don't really know how it renders on different devices, so having an input from you would be awesome

**Review by johanstokking — APPROVED** [review]
@nejraselimovic props, this is a good solution

**Comment by kschiffer** [comment]
I suggest to close this PR and merge it into #145, while getting rid of setting relative width values.

**Pull request closed without merging**